client/helpers/index.js

```diff
@@ -75,3 +76,12 @@ export const flattenReveals = (fields, values) => {
     ])
   }, []);
 };
+
+export const changed = (fields, latest, granted) => {
+  if(fields.some(k=> latest.some(l=> l.includes(k)))) {
+    return LATEST;
+  }
+  if(fields.some(k=> granted.some(l=> l.includes(k)))) {
+    return GRANTED;
+  }
+}
```

Comment:
Not sure this is the right place for this "helper" - it is only used once and feels like it should be just defined where it is used - I would suggest creating a <Changed /> or <HasChanged /> or something component and using this wherever we show a changed label